### DISCUSSION TITLE
n-of-m/multisig encryption

### DISCUSSION THREAD

**Discussion by danielschonfeld** [FiloSottile/age#305] [General]
Are there any plans to add the ability to encrypt/decrypt with an n-of-m keys scheme?

Thanks!

**Comment by lildata** [comment]
That would be great.
Right now you can do this with SOPS https://github.com/mozilla/sops which support age.

**Comment by str4d** [comment]
This would be a perfect candidate for an age plugin!

**Comment by covert-encryption** [comment]
Out of curiosity, if you may share, what is your use case for this? Given that whoever encrypts the message still can open it alone (or just keep the content in its plaintext form otherwise), it is hard to see where multisig would be useful.

**Comment by olastor** [comment]
I tried to implement this as a plugin in https://github.com/olastor/age-plugin-sss . Compared to SOPS, you can also use recursive / more complex splitting structures.

**Reply by FiloSottile** [reply]
Ooooh I like the `--generate-recipient` and `--generate-identity` strategies for turning arbitrary config files into plugin recipients/identities!

**Reply by olastor** [reply]
Thanks!
